_commands/basics/comm.md

```diff
@@ -0,0 +1,45 @@
+---
+---
+
+comm
+---
+Compares two sorted files line by line and writes three columns to standard output.
+<!-- one line explanation would go here -->
+
+<!-- minimal example -->
+~~~ bash
+cat words1.txt
```

Comment:
What do the three columns mean? This isn't explained anywhere in the information given.